src/palette-panel/components/UIElementsTree.tsx

```diff
@@ -0,0 +1,127 @@
+/**
+ * ---------------------------------------------------------------------
+ * Copyright (c) 2020 EclipseSource Munich
+ * Licensed under MIT
+ * https://github.com/eclipsesource/jsonforms-editor/blob/master/LICENSE
+ * ---------------------------------------------------------------------
+ */
+import { Layout, UISchemaElement } from '@jsonforms/core';
+import Collapse from '@material-ui/core/Collapse';
+import {
+  createStyles,
+  fade,
+  makeStyles,
+  withStyles,
+} from '@material-ui/core/styles';
+import { TransitionProps } from '@material-ui/core/transitions';
+import Typography from '@material-ui/core/Typography';
+import HeightIcon from '@material-ui/icons/Height';
+import TreeItem, { TreeItemProps } from '@material-ui/lab/TreeItem';
+import TreeView from '@material-ui/lab/TreeView';
+import React from 'react';
+import { useDrag } from 'react-dnd';
+import { animated, useSpring } from 'react-spring/web.cjs'; // web.cjs is required for IE 11 support
+
+import { DndItems } from '../../core/dnd';
+
+const useStyles = makeStyles(
+  createStyles({
+    root: {
+      flexGrow: 1,
+      maxWidth: 400,
+    },
+  })
+);
+
+const TransitionComponent = (props: TransitionProps) => {
```

Comment:
As this is a copy of the other tree we should extract common elements like this TransitionComponent.